src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
-body {
+@mixin flex-space {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+$grey: #616070;
+$blue-accent: #00acdc;
+$main-accent: #060b35;
+$white: #fff;
+
+* {
+  font-family: "Roboto", sans-serif;
```

Comment:
Don't use for `*` selector any rules except of `border-box`. 
You can move these rules to the `body` selector